peer/core.yaml

```diff
@@ -335,6 +335,13 @@ ledger:
       # Deploy chaincodes into the genesis block
       chaincodes:
 
+        noop_syscc:
+          path: github.com/hyperledger/fabric/core/system_chaincode/noop
+          type: GOLANG
+          constructor:
+            args:
+
+
```

Comment:
@ghaskins it was a fallacy to assume any protection of data on blockchain. If protection is needed for a data then it should not be on block chain or should be encrypted.  Relying on access control is not a part of this technology.